movies-hub/src/pages/Login.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react";
+import { Eye, EyeOff } from "lucide-react";
+import { successToast, warningToast, errorToast } from "../utils/DisplayToast";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [error, setError] = useState("");
+  const apiLoginUrl = import.meta.env.VITE_API_LOGIN_URL;
+  const apiKey = import.meta.env.VITE_API_KEY;
+  const apiToken = import.meta.env.VITE_API_ACCESS_TOKEN;
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (!email || !password) {
+      setError("Please enter both email and password.");
+      return;
+    }
+
+    try {
+      setError("");
+      const response = await fetch(apiLoginUrl, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ email, password }),
+      });
```

Comment:
Do with axios and then move it to a file api.js

First make supporting file axiosInstance.js